src/components/UserTableItem.tsx

```diff
@@ -0,0 +1,32 @@
+import { Link } from "react-router-dom";
+import { UserData } from "types";
+
+const UserTableItem = ({
+    id,
+    lastname,
+    name,
+    surname,
+    removeUser,
+    getVisitUrl,
+}: UserData & {
+    removeUser: (id: number) => void;
+    getVisitUrl: (id: number) => string;
+}) => {
+    return (
+        <tr key={id}>
```

Comment:
no need for key here
```suggestion
        <tr>
```